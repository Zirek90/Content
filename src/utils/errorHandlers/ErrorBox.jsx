import React from "react";
import { ErrorContainter } from "../../styles/ContainerStyles";
import { ErrorBtn } from "../../styles/ButtonStyles";

export const ErrorBox = () => (
  <ErrorContainter>
    <p>Ops seems like there is some problem. Please reload page</p>
    <ErrorBtn onClick={() => window.location.reload()}>Reload page</ErrorBtn>
  </ErrorContainter>
);
