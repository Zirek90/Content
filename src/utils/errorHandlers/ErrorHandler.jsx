import React from "react";
import { ErrorExceptionWrapper } from "../../styles/ContainerStyles";

export const ErrorHandler = () => (
  <ErrorExceptionWrapper role="alert">
    <p>Could not download content</p>
  </ErrorExceptionWrapper>
);
