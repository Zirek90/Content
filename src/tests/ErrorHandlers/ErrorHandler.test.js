import React from "react";
import renderer from "react-test-renderer";
import { ErrorHandler } from "../../utils/errorHandlers/ErrorHandler";

describe("ErrorHandler", () => {
  it("renders correctly", () => {
    const tree = renderer.create(<ErrorHandler />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
