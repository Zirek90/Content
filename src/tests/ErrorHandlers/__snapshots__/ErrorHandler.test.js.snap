// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ErrorHandler renders correctly 1`] = `
<div
  className="sc-gKAblj ifuKnE"
  role="alert"
>
  <p>
    Could not download content
  </p>
</div>
`;
